mobile/build.gradle.kts

```diff
@@ -34,7 +34,12 @@ android {
     targetCompatibility = JavaVersion.VERSION_1_8
   }
 
-  testOptions { packagingOptions { jniLibs { useLegacyPackaging = true } } }
+  testOptions {
+    packagingOptions { jniLibs { useLegacyPackaging = true } }
+
+    // Allow us to have Log during testing
+    unitTests.isReturnDefaultValues = true
```

Comment:
AFAIK this won't let us use `Log` during testing, but instead not let the code crash if we run unit tests and use `Log.***` statements in the speech recogniser.

The docs indicate that this should only used as a last resort (https://developer.android.com/training/testing/local-tests#error); what would you think of using some `println()` statements instead of `Log.i` so the tests don't fail, and so that we don't change the behaviour of all the other framework methods in our unit tests ?

Reply:
Fix in https://github.com/epfl-SDP/android/pull/311/commits/50e3bafde70780da31e8f05a6e97223123b869ce